Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -557,11 +669,11 @@ extension SwiftLanguageServer {
           selectionRange = range
         }
 
-        let children: [DocumentSymbol]?
+        let children: [DocumentSymbol]
         if let substructure: SKDResponseArray = value[self.keys.substructure] {
           children = documentSymbols(array: substructure)
         } else {
-          children = nil
+          children = []
         }
```

Comment:
Instead of updating the test, I have updated the `documentSymbol` request implementation instead to always return an array of children. This feels cleaner to me and would also reflect the previous behavior (we never actually entered the `else` case).

What do you think?

Reply:
SGTM.